tests/generate_packageTest.py

```diff
@@ -136,12 +146,14 @@ def test_register(self, call_mock):
 
     def test_no_bear_object(self):
         if not os.path.exists(self.NO_BEAR_PATH):
-            os.makedirs(os.path.join('bears', 'BadBear'))
+            os.makedirs(os.path.join(self.BEARS_PATH, 'BadBear'))
             touch(self.NO_BEAR_PATH)
         main()
         self.assertFalse(os.path.exists(os.path.join(
-            'bears', 'upload', 'BadBear')))
-        shutil.rmtree(os.path.join('bears', 'BadBear'))
+            self.BEARS_UPLOAD_PATH, 'BadBear')))
+        shutil.rmtree(os.path.join(self.BEARS_PATH, 'BadBear'))
 
     def tearDown(self):
-        shutil.rmtree(os.path.join('bears', 'upload'))
+        shutil.rmtree(os.path.join(self.BEARS_UPLOAD_PATH))
+
+        os.chdir(self.orig_dir)
```

Comment:
This is the entire purpose of this patch.

The problem with https://github.com/coala/coala-bears/issues/1235 was that the tests were run in the `coala-bears` directory.

I'm trying to run the tests outside the coala-bears directory, which is finding bugs like this one, and the MarkdownBear bug, and will in the future prevent them from happening.